### ISSUE TITLE
Not an Issue, but just need help changing the form background color

### ISSUE THREAD

**Issue by jumper77** [thielj/MetroFramework#72]
I just started using the product, but I'm having a hard time getting the form background any other color than white. I have the style manager included and it's set to dark theme. But form remains white.

Any help greatly appreciated.

**Comment by aidabeorn** [comment]
in your form.designer.cs, within IntiializeComponent(), add `this.theme = MetroThemeStyle.Dark;`

**Comment by jumper77** [comment]
Thank you so much. I'm was getting ready to ask a new question, but maybe you could help, if you don't mind. In Visual Studio, whenever I mouse over any MaterialSkin control, the cursor turns in the the "MoveAll" cursor. This only started happening a short while ago. I tested it in another application and the results were the same.

Have you ever heard about this happening?
thanks,
john

**Comment by aidabeorn** [comment]
Nope, sorry, I don't use VS that much, try asking on SO or similar

**Comment by jumper77** [comment]
That's ok. I fixed the issue not long after we talked. Thanks for your help.
